templates/shared/forms/interactive/container.html

```diff
@@ -0,0 +1,187 @@
+<div class="p-modal" id="contact-modal">
+  <div class="p-modal__dialog is-paper"
+       role="dialog"
+       aria-labelledby="modal-title"
+       aria-describedby="modal-description">
+    <div class="js-pagination js-pagination--1">
+      <div class="js-formfield">
+        <div class="row--50-50 p-strip is-shallow">
+          <div class="col">
+            <h2 class="p-heading--1" id="modal-title">Get in touch with the container team</h2>
+            <p>Fill in this form and a member of our team will be in touch shortly.</p>
+          </div>
+          <div class="col">
+            <button class="p-modal__close js-close" aria-label="Close active modal">Close</button>
+            <form action="/marketo/submit"
+                  method="post"
+                  id="mktoForm_%% formid %%"
+                  class="modal-form">
+              <div class="p-section--shallow">
+                <label for="firstName">First name:</label>
```

Comment:
```suggestion
                <label for="firstName" class="is-required">First name:</label>
```
Please add to all required fields, vanilla docs [here](https://vanillaframework.io/docs/base/forms#required)